src/TwigComponent/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.25.0
+
+-  Fix `Template#loadTemplate` deprecation for Twig >=3.21
```

Comment:
Not something that user should be noticed in the Changelog imho

Reply:
Ah? I believe people will be happy to know the deprecation is fixed:
> as we are getting more and more complaints in support channels about Symfony triggering Twig deprecations

More generally, I don't like not seeing a changelog when a package tags a new version. I don't know what's been changed, and it may look suspicious. 

Maybe the actual entry is too technical? What about "Fix deprecations with Twig >= 3.21"?

Reply:
That's not the same as the changelog we publish / visible on the website.

I think the Symfony standard is to only write what can change / been used by the user (new features mainly) 

(imagine a changelog with all the bug fixes, you don't see anything anymore)

Reply:
I've just looked at ~10 random fixes PR on symfony/symfony, and indeed none of them updated the changelog.

I will remove the entry then :)